<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Malles</class>
 <widget class="QWidget" name="F_Malles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QCommandLinkButton" name="Bt_EmprunterMalle">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Emprunter</string>
       </property>
       <property name="icon">
        <iconset resource="ImagesIcones.qrc">
         <normaloff>:/Panier.png</normaloff>:/Panier.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCommandLinkButton" name="Bt_SupprimerMalle">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
       <property name="icon">
        <iconset resource="ImagesIcones.qrc">
         <normaloff>:/Supprimer.png</normaloff>:/Supprimer.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="CBx_Annee">
     <item>
      <property name="text">
       <string>2018</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2019</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2020</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2021</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2022</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2023</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2025</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2026</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2027</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2028</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2029</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2030</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="CBx_Mois">
     <item>
      <property name="text">
       <string>Janvier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Février</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mars</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Avril</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mai</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Juin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Juillet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Août</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Septembre</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Octobre</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Novembre</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Décembre</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gL_CalendrierMalles">
     <item row="0" column="0">
      <widget class="QLabel" name="fake_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
